src/Select.js

```diff
@@ -320,6 +323,11 @@ const Select = React.createClass({
 		}
 	},
 
+	handleRequired (value, multi) {
+		if (!value) return true;
+		return (multi ? value.length === 0 : Object.keys(value).length === 0);
```

Comment:
@dmatteo @JedWatson Why check if a single value has any keys here? It breaks when the value is a number.